cms/templates/cms/toolbar/toolbar_javascript.html

```diff
@@ -31,9 +31,10 @@
             'error': '<strong>{% filter escapejs %}{% trans "The following error occured:" %}{% endfilter %}</strong> ',
             'success': '{% filter escapejs %}{% trans "Action successfull... reloading." %}{% endfilter %}',
             'confirm': '{% filter escapejs %}{% trans "Are you sure you want to delete this plugin?" %}{% endfilter %}',
-            'confirmEmpty': '{% filter escapejs %}{% trans "({1}) plugin is empty. Remove it?" %}{% endfilter %}',
             'publish': '{% filter escapejs %}{% trans "Are you sure you want to publish this page?" %}{% endfilter %}',
-            'addPluginPlaceholder': '{% filter escapejs %}{% trans "Plugin will be added here" %}{% endfilter %}'
+            'addPluginPlaceholder': '{% filter escapejs %}{% trans "Plugin will be added here" %}{% endfilter %}',
+            'confirmDirty': '{% filter escapejs %}{% trans "There are some unsaved changes." %}{% endfilter %}',
+            'confirmDirtyESC': '{% filter escapejs %}{% trans "Are you sure you want to stop editing?" %}{% endfilter %}'
```

Comment:
discussed this separately, not possible to affect buttons, result: "Are you sure you want to abandon these changes?"